lib/engine/corporation.rb

```diff
@@ -33,6 +33,7 @@ def initialize(sym:, name:, tokens:, **opts)
       @trains = []
       @companies = []
       @revenue_history = {}
+      @last_run = nil
```

Comment:
can we store all the runs and replace revenue history with this

Reply:
yeah, but we need to store the connection_hexes in textual form due to upgrades on tiles.

I'll add a new class to store this info and replace it with @operating_history

Reply:
you can store both the text form and maybe a property for revenue

Reply:
yep, see the diff :)